Dockerfile

```diff
@@ -31,6 +31,8 @@ RUN ./manage.py collectstatic --noinput
 RUN apt-get clean && apt-get purge
 USER mitodl
 
+RUN mkdir /var/media && chown -R mitodl:mitodl /var/media
```

Comment:
We can definitely try that, but I want to spend some time and see if we can get to the bottom of this issue. It would help to know what the difference is, here.